libs/ha/ha_common_libs_k8s.py

```diff
@@ -1907,3 +1929,45 @@ def form_endpoint_port(pod_obj, pod_list, port_name="rgw-https"):
             pod_ep_dict[pod] = ip_port
 
         return pod_ep_dict
+
+    def create_iam_user_with_header(self, i_d, header):
```

Comment:
Instead of i_d, we should pass num_users. And we can create users in loop.

Reply:
> Instead of i_d, we should pass num_users. And we can create users in loop.

But then we have to change logic in main loop,

Reply:
Please make changes as discussed.